### PR TITLE
DB-10615 Support known keywords as tokens instead of string in EXPORT 

### PR DESCRIPTION
**Pull request by arnaud-splice**

### CODE REVIEW

**Comment by spliceci** [comment]
TEST SUCCEEDED +1

File: splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/export/ExportOperationIT.java [hunk 1]
```diff
@@ -536,9 +544,9 @@ private String buildExportSQL(String selectQuery, String exportPath, String comp
         if (useNativeSyntax) {
             StringBuilder sql = new StringBuilder();
             sql.append("EXPORT TO '").append(exportPath).append("'");
-            sql.append(" AS 'csv' ");
```
Comment (hatyo) [inline splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/export/ExportOperationIT.java:539]: but wouldn't this remove the tests for exporter checking the syntax internally (not the parser)?

**Review by martinrupp — APPROVED** [review]
nice

**Review by hatyo — COMMENTED** [review]
Nice work! It would be nice to have a couple of tests checking that the new syntax works (`EXPORT TO '/tmp/' AS CSV COMPRESSION SNAPPY`) as well as the old syntax (`EXPORT TO '/tmp/' AS 'CSV' COMPRESSION 'SNAPPY'`)

**Review by hatyo — APPROVED** [review]
great!

**Review by carolp-503 — APPROVED** [review]
+1

**Pull request merged**